drivers/pinmux/stm32/pinmux_board_olimex_stm32_e407.c

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
@nashif: I'm working on moving all STM32 pinmux_board_.... from pinmux drivers to boards/*.
I would prefer that until my is upstreamed, all stm32bord ports adhere to the existing stm32 porting so I have only one kind of implementation to take care of.
Is that ok with you to let the current porting, going on, remove your comment and let me do the pinmux > board transition?